### PR TITLE
Fix Round

### PR DESCRIPTION
**Pull request by tpreusse**
Changes
- allow custom pay notes trial form in documents on iOS
- [disable FLoC](https://twitter.com/natfriedman/status/1387159870667849731)
- nicer spacing on pages with custom pay notes

### Spacing

| before                                                                                                                                                           | after                                                                                                                                                               |
|------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| screencapture-republik-ch-probelesen-2021-04-30-12_12_01 | screencapture-localhost-3010-probelesen-2021-04-30-12_10_43 |

(couldn't find a case where the br where necessary with pages like about and statutes)

### CODE REVIEW

File: components/Article/PayNote.js [hunk 1]
```diff
@@ -235,12 +235,24 @@ const generateKey = (note, index) => {
   return { ...note, key: `custom-${index}` }
 }
 
+const isPermissibleIOSCta = cta => !cta || cta === 'trialForm'
```
Comment (annatraussnig) [inline components/Article/PayNote.js:238]: wouldn't `cta !== 'button'` be less convoluted?

**Review by annatraussnig — APPROVED** [review]
LGTM.

File: components/Article/PayNote.js (hunk 1 above)
Comment (tpreusse) [reply components/Article/PayNote.js:238]: Would be shorter but generally I prefer declaring what is allowed: no cta and `trialForm`. IMO easier to understand what the policy is.

**Comment by tpreusse** [comment]
Thanks for the review 🙇

**Pull request merged**